server/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -3042,4 +3042,25 @@ public void testSubscriptExpressionFromUnionAll() throws Exception {
         assertThat(analyzed.outputs().size(), is(1));
         assertThat(analyzed.outputs().get(0), isFunction("subscript", isField("obj"), isLiteral("unknown")));
     }
+
+    @Test
+    public void test_declare_is_not_supported() {
+        var executor = SQLExecutor.builder(clusterService).build();
```

Comment:
Also not part of the PR, could we extract the executor and make ie available for all (or at least most) tests?

Reply:
Can do it in the follow up PR that adds the implementation. 

In general I kinda prefer the pattern of making it local to the test as it gives you more flexibility defining the tables without adding overhead due multiple table definitions. But I guess in this case it's fine to have a single setup

Reply:
Not insisting, I was mostly asking if there is a reasoning/benefit of this pattern.